zio-http/src/main/scala/zio/http/ChannelEvent.scala

```diff
@@ -70,4 +49,19 @@ object ChannelEvent {
     case object HandshakeTimeout  extends UserEvent
     case object HandshakeComplete extends UserEvent
   }
+
+  def channelRead[A](msg: A): ChannelEvent[A] =
```

Comment:
Is the `channel*` prefix adding anything to these constructors? Also it's inconsistent since it's not being used for `exceptionCaught` and `userEventTriggered`.